src/main/java/dev/joopie/jambot/repository/artist/ArtistFinder.java

```diff
@@ -0,0 +1,25 @@
+package dev.joopie.jambot.repository.artist;
+
+import dev.joopie.jambot.models.Artist;
+import dev.joopie.jambot.repository.base.BaseFinder;
+import io.ebean.DB;
+
+import java.util.List;
+
+public class ArtistFinder extends BaseFinder<Artist> {
+    private static final String NAME = "name";
```

Comment:
Niet logischer om dit soort velden in Artist te zetten? Het zijn tenslotte de velden van die entity